src/ds/json/create.md

```diff
@@ -0,0 +1,76 @@
+## Create JSON documents
+
+Here's a query that creates a JSON document describing a single bike.
+
+```redis Create a JSON document
```

Comment:
Do we need this long text? I guess it is useful when explaining RediSearch, but I think we can use something shorter here.

Reply:
Shortened (next push).